README.md

```diff
@@ -92,3 +92,14 @@ A [token-bucket](https://en.wikipedia.org/wiki/Token_bucket) rate limiter implem
 
   limiter.take # true
 ```
+
+# Configuration
+
+We use a connection pool, because a connection shared among multiple threads
+will produce unexpected results when using blocking Redis operations.
```

Comment:
"will produce unexpected results" sounds to me like "will affect correctness," not "will affect performance." You could use the latter phrase.